translib/transformer/xfmr_intf.go

```diff
@@ -1361,8 +1447,38 @@ var intf_subintfs_table_xfmr TableXfmrFunc = func(inParams XfmrParams) ([]string
 	var tblList []string
 
 	pathInfo := NewPathInfo(inParams.uri)
+	ifName := pathInfo.Var("name")
 	idx := pathInfo.Var("index")
 
+	if inParams.oper == SUBSCRIBE {
+		var _intfTypeList []E_InterfaceType
+
+		if idx == "*" || idx != "0" {
```

Comment:
Shouldn't it be `if idx != "*" && idx != "0" {/* error */}`. Current condition rejects the wildcard key and hence blocks traget_defined subscription at a higher level like interface list

Reply:
Thanks for reviewing the PR @sachinholla. I have addressed the comments. Please review.